content/400-pulse/100-what-is-pulse.mdx

```diff
@@ -6,23 +6,48 @@ tocDepth: 3
 toc: true
 ---
 
-<TopBlock>
+## What is Prisma Pulse?
 
-[Prisma Pulse](https://www.prisma.io/data-platform/pulse) is managed database-event infrastructure that captures and distributes your database events to your application. It simplifies subscribing to type-safe data changes with an extended [Prisma Client](/orm/prisma-client) to power real-time functionality.
+[Prisma Pulse](https://www.prisma.io/data-platform/pulse) is managed infrastructure that captures databases events (i.e. create, update and delete operations) and distributes them to your applications.
 
-</TopBlock>
+It lets you subsribe to any changes happening in your database using [Prisma Client](/orm/prisma-client) to power **real-time use cases** in your applications, such as:
 
-## How Prisma Pulse works
+- chat and messaging
+- data syncing (e.g. into a search index)
+- social interaction and collaboration
+- notifying users
+- updating inventories
+- managing payments
+- ... and any other functionality that requires real-time updates
 
-Prisma Pulse leverages Change Data Capture (CDC) to efficiently observe and capture database changes as they occur. By monitoring the database's transaction log, Prisma Pulse identifies change events like inserts, updates, and deletes without impacting the database's performance.
+Prisma Pulse integrates with [Prisma ORM](/orm) and easily lets you subscribe database changes in a type-safe way using Prisma Client:
 
-The captured events are processed, evaluated, and swiftly distributed to relevant client subscriptions ensuring your applications stay synchronized with the latest database state.
+![](/img/pulse/overview.png)
```

Comment:
I think a lot of this image could instead be an ordered list and code snippets, plus some smaller parts of the visual, which would flow better with the docs page. What do you think?

Reply:
Hmm, I prefer the power of a good visual (of which I think we generally have way too little in the docs).

But I see how this can be turned into text-based content where essentially only the illustration of step 2. is needed because everything else is text and code snippets?

I would probably still not use a bullet list but rather use numbered headings like:

### 1. Subscribe

Subscribe to changes on a model with Prisma Client, e.g. `User`.

```
// subscribe code
```

### 2. Database event jhappens

A change is made ...

![](insert.png)

### 3. Database event is propagated

Prisma Pulse propages ....

```
// terminal output
```

--- 

Do you prefer that remove the illustration that depicts the whole flow and turn it into this text-based version?

Reply:
> But I see how this can be turned into text-based content where essentially only the illustration of step 2. is needed because everything else is text and code snippets?
I would probably still not use a bullet list but rather use numbered headings like:

Yep, you get it!

I personally would prefer it, because I think it's more readable, more easily updateable, and can still well integrate smaller visuals. But up to you if you feel strongly about it.

Reply:
I went with that approach for now because I don't have a vertical layout yet: https://13a1647e.docs-51g.pages.dev/pulse/what-is-pulse

<img width="338" alt="image" src="https://github.com/prisma/docs/assets/4058327/643f6bde-3dee-4c0b-9a65-5a2ec99e2451">

I think it works but what's a bit weird is that the illustration is so huge compared to the text.

Reply:
Got it fixed with Alejandro: 

<img width="552" alt="image" src="https://github.com/prisma/docs/assets/4058327/f142cc46-8c33-4c39-9f5f-a2acedb3d057">

I think the PR should be ready to merge now!